pennylane/gradients/vjp.py

```diff
@@ -115,16 +115,8 @@ def compute_vjp_single(dy, jac, num=None):
     if not isinstance(dy_row, np.ndarray):
         jac = _convert(jac, dy_row)
 
-    try:
-        if _all_close_to_zero(dy):
-            # If the dy vector is zero, then the
-            # corresponding element of the VJP will be zero.
-            num_params = len(jac) if isinstance(jac, tuple) else 1
-
-            res = qml.math.convert_like(np.zeros(num_params), dy)
-            return qml.math.cast_like(res, dy)
-    except (AttributeError, TypeError):
-        pass
+    # Note: For generality, all exception type warnings are disabled.
+    # TODO: Excplictly catalogue and update raises for known types.

```

Comment:
Is this referring to the try/except blocks? From this text, I was expecting to see us filtering out certain warnings.

Reply:
Yes, in relation to the exceptions. For now, we can catch all, and fallback to older behaviour if any failure appears (majority of cases will work fine within the try block). For all others, we'd need a list of the potential failures, so catching all and avoiding pylint raising a warning is sufficient for now.

Reply:
Got it! This is very optional but could the following be a good explanation:
```suggestion
    # Note: The following uses try/except blocks to attempt a fast
    # dispatch before falling back to a slower method. For
    # simplicity, all exceptions are handled together.
    # TODO: Explicitly catalogue and update raises for known types.
```